Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts

```diff
@@ -28,10 +28,10 @@ app.use(function (req, res, next) {
 // will print stacktrace
 if (app.get('env') === 'development') {
     app.use((err: any, req, res, next) => {
-        res.status(err['status'] || 500);
+        res.status(err.status || 500);
         res.render('error', {
+            error: err,
```

Comment:
yes. I ran the templates through tslint/eslint respectively with moderately aggressive rules.

In this particular case, I believe it was a matter of alphabetizing object literal members.